js/style/style_layer/fill_extrusion_style_layer.js

```diff
@@ -4,6 +4,15 @@ const StyleLayer = require('../style_layer');
 const FillExtrusionBucket = require('../../data/bucket/fill_extrusion_bucket');
 
 class FillExtrusionStyleLayer extends StyleLayer {
+
+    getPaintValue(name, globalProperties, featureProperties) {
+        const value = super.getPaintValue(name, globalProperties, featureProperties);
+        if (name === 'fill-extrusion-color') {
+            value[3] = 1;
```

Comment:
Looking at the vertex shader, it seems that `fill-extrusion-color` alpha value is not used there at all. So we could simply remove this code. @lbud does this sound right?